lib/ui/text.dart

```diff
@@ -2,7 +2,8 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-// @dart = 2.6
+// @dart = 2.9
```

Comment:
Shouldn't this be removed now?

Reply:
Until I put this something in the pipeline wasn't happy. I'll try removing it after I address comments.